packages/server/src/fhir/routes.test.ts

```diff
@@ -587,4 +587,39 @@ describe('FHIR Routes', () => {
         .send({ ...membership, accessPolicy: undefined });
       expect(res3.status).toBe(403);
     }));
+
+  test('Search multiple resource types with _type', async () =>
+    withTestContext(async () => {
+      const { accessToken } = await createTestProject({ withAccessToken: true });
+
+      const res1 = await request(app)
+        .post('/fhir/R4/Patient')
+        .set('Authorization', 'Bearer ' + accessToken)
+        .send({ resourceType: 'Patient' });
+      expect(res1.status).toBe(201);
+
+      const res2 = await request(app)
+        .post('/fhir/R4/Observation')
+        .set('Authorization', 'Bearer ' + accessToken)
+        .send({
+          resourceType: 'Observation',
+          status: 'final',
+          code: { text: 'test' },
+          subject: { reference: `Patient/${res1.body.id}` },
+        });
+      expect(res2.status).toBe(201);
+
+      const res3 = await request(app)
+        .get('/fhir/R4?_type=Patient,Observation')
```

Comment:
Actually is `/fhir/R4/?_type=Patient,Observation` also supported here?

Reply:
Ah, the trailing slash?  Good question.  I'll investigate

Reply:
Confirmed, works.  Spec is ambiguous.  I added a test to lock it in with this behavior.